packages/@okta/vuepress-site/docs/guides/pwd-optional-sign-in-email/main/react/integrationsummary.md

```diff
@@ -0,0 +1,5 @@
+<div class="full">
+
+![Sequence diagram illustrating the password-optional sign-in use case.](/img/pwd-optional/pwd-optional-sign-in-js-react-summary.png)
```

Comment:
Can we remove the word "new" from the chevron in step 3 please?  
It reads "OIE is requesting the verification of a new email" rather than "OIE is requesting a new verification via an old email" which is what is happening as this is a sign-in flow rather than a sign-in flow or a change-your-password flow

```suggestion
### 3. Identity Engine requests sends user email challenge
```

Reply:
Changed to Identity Engine requests email verification